dockerfile_scripts/tf2.4-cuda11.1-symlink.sh

```diff
@@ -0,0 +1,5 @@
+#!/usr/bin/env bash
+
+set -e
+
+ln -s /usr/local/cuda/lib64/libcusolver.so.11 /opt/conda/lib/python3.7/site-packages/tensorflow/python/libcusolver.so.10
```

Comment:
this is a style preference, but I'd inline such one-liners directly into dockerfile. a separate file feels redundant and doesn't improve readability that much.

Reply:
good point. created it thinking there would be more symlinking needed but this is unnecessary now. removed